prow/config.yaml

```diff
@@ -3844,6 +3844,26 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  kubernetes-incubator/service-catalog:
+  - name: svc-cat-test
+    agent: kubernetes
+    context: svc-cat-test
+    always_run: true
```

Comment:
you might want to add `skip_report: true` so this runs on every PR still, but doesn't post a status or any comments. we can check on prow.k8s.io / k8s-gubernator.appspot.com for results while we make sure this behaves properly, then remove the `skip_report`.